types/lodash/index.d.ts

```diff
@@ -13908,10 +13908,30 @@ declare namespace _ {
          * _.pick(object, ['a', 'c']);
          * // => { 'a': 1, 'c': 3 }
          */
-        pick<T extends object>(
+        pick<T extends object, U extends keyof T>(
+            object: T | null | undefined,
+             ...props: U[]
+        ): Pick<T, U>;
+
+        pick<T extends object, U extends keyof T>(
+            object: T | null | undefined,
+            props: U[]
+        ): Pick<T, U>;
+
+        pick<T>(
+            object: T | null | undefined,
+            props: PropertyPath[]
+        ): PartialObject<T>;
+
+        pick<T>(
             object: T | null | undefined,
             ...props: PropertyPath[]
         ): PartialObject<T>;
+
+        pick<T extends object, U extends keyof T>(
+            object: T | null | undefined,
+            ...props: PropertyPath[]
+        ): Pick<T, U>;
     }
```

Comment:
Why do we need this overload?

Reply:
`U` is not used except in the return value. This overload is basically the same as the previous one, except this one violates  the [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes). I think this overload should be removed.

Reply:
Yup, we don't need this overload. I'll delete this one